mlocal/frags/Makefile.stub

```diff
@@ -77,9 +82,8 @@ test:
 	@echo " TEST sudo go test [all]"
 	$(V)M=0; eval 'while [ $$M -le 20 ]; do sleep 60; M=`expr $$M + 1`; echo "Still testing ($$M) ..."; done &' ; \
     	trap "kill $$! || true" 0; \
-		cd $(SOURCEDIR) && \
-		sudo -E \
-		scripts/go-test -v -tags 'integration_test' \
+		cd $(SOURCEDIR) && sudo -v && \
```

Comment:
the bit with "sudo -v" is already in the sudo wrapper you are adding...

Reply:
yep, ideally that should go in `scripts/go-test`